src/components/implementation/fork_tests/malloc_comp/Makefile

```diff
@@ -0,0 +1,8 @@
+C_OBJS=malloc_comp.o
+ASM_OBJS=
+COMPONENT=malloccomp.o
+INTERFACES=test_malloc_comp
+DEPENDENCIES=cbuf_mgr printc sched lock valloc # all of these??? Probably not
```

Comment:
Please include only dependencies that are required.  If you don't know what's required, then you don't know which code your running.  That's the key problem.